docker/Dockerfile

```diff
@@ -30,6 +30,39 @@ COPY firefly/package.json firefly/yarn.lock ./
 RUN yarn install --ignore-platform --frozen-lockfile
 
 
+FROM node_module AS dev_env
+
+RUN apt-get update \
+# install tomcat9 for dev mode \
+    && curl -O https://dlcdn.apache.org/tomcat/tomcat-9/v9.0.64/bin/apache-tomcat-9.0.64.tar.gz \
+    && mkdir /opt/tomcat && tar xzvf apache-tomcat-9*tar.gz -C /opt/tomcat --strip-components=1 \
+    && sed -i -E 's/allow=".*"/allow=".*"/' /opt/tomcat/webapps/manager/META-INF/context.xml \
+# cleanup
+    && rm apache-tomcat-9*tar.gz \
+    && rm -rf /var/lib/apt/lists/*;
+
+ENV JPDA_ADDRESS=*:5050 \
+    CATALINA_HOME=/opt/tomcat \
+    DEBUG=true \
+    START_MODE=start \
+    USE_ADMIN_AUTH="false"
+
+ENV work_dir=firefly
+ENV project=firefly
+
+WORKDIR ${CATALINA_HOME}
+
+# Tomcat scripts, config files, etc
+COPY firefly/docker/*.sh .
+COPY firefly/docker/tomcat-users.xml conf/
+
+# set up directory protections, copy stuff around, add tomcat user and group
+RUN mkdir -p /firefly/workarea /firefly/logs/statistics /firefly/alerts \
+    && chmod a+x *.sh
+
+CMD /opt/tomcat/launchTomcat.sh && cd /opt/work/${work_dir} && gradle ${project}:bAD && gradle ${project}:dev
```

Comment:
This appears that it will honor all of the `PROPS_*` environment variables. right?

Reply:
It should.  I tried to use the same startup script so that DevMode is as close to the normal deployment as possible.